treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
load_initial_data management command now instead of 'load_preseed' will call 'reset_seta' command which ensures JobPriority table is empty and then it calls 'analyze_failures' and 'load_preseed' which prevents creation of rows with 2 weeks expiration time.